nbs/sample_question.ipynb

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
<p>This example looks Really good now.</p><p>I would just add some examples in the exposition so that the task is clear.</p><p>Maybe print the example text and say what you would like to deduce for example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>